library/build.gradle

```diff
@@ -26,6 +26,7 @@ dependencies {
   api rootProject.ext.recyclerView
 
   testImplementation 'junit:junit:4.12'
+  testImplementation 'org.robolectric:robolectric:3.8'
```

Comment:
Why do we need Robolectiric? Is there no other way around this? I would like to keep Robolectric out of this library. It causes additional maintainance costs and often it indicates some API flaws that we might could solve properly in a different way.